drivers/ethernet/Kconfig.mcux

```diff
@@ -7,6 +7,8 @@
 menuconfig ETH_MCUX
 	bool "MCUX Ethernet driver"
 	depends on HAS_MCUX
+	select NOCACHE_MEMORY if CPU_CORTEX_M7
```

Comment:
Minor nitpick, you might now want to use `HAS_MCUX_CACHE` instead of `CPU_CORTEX_M7`. Otherwise it looks all good.